src/function/aggregate/holistic/mode.cpp

```diff
@@ -0,0 +1,178 @@
+// MODE( <expr1> )
+// Returns the most frequent value for the values within expr1.
+// NULL values are ignored. If all the values are NULL, or there are 0 rows, then the function returns NULL.
+
+#include "duckdb/common/exception.hpp"
+#include "duckdb/common/vector_operations/vector_operations.hpp"
+#include "duckdb/function/aggregate/holistic_functions.hpp"
+#include "duckdb/planner/expression/bound_aggregate_expression.hpp"
+
+#include "duckdb/common/unordered_map.hpp"
+
+#include <functional>
+
+namespace std {
+
+template <>
+struct hash<duckdb::interval_t> {
```

Comment:
Could this be moved to `common/hash.hpp`?

Reply:
I thought about that originally, but they are *different* hashes!  One is a 64 bit hash we use internally; the other is what the standard library uses.  Which is why they are in `namespace std`.

Reply:
Fair enough, then I will merge the PR as is. Thanks again!